pages/report/[reportId].tsx

```diff
@@ -19,10 +21,16 @@ export const ReportById = () => {
     <ReportPage
       report={toReport(state.currentReport)}
       onDeleteFile={(_, file) => {
-        if (file) filePloc.delete(file);
+        if (file) {
+          filePloc.delete(file)
+          handleToast('File deleted')
+          // handleFileState
```

Comment:
What is this comment? If it is something that is pending it should have more information or be removed.